marathon_acme/cli.py

```diff
@@ -52,6 +52,12 @@
                     help='The address for the port to listen on (default: '
                          '%(default)s)',
                     default=':8000')
+parser.add_argument('--sse-timeout',
+                    help=('Amount of time in seconds to wait for some event '
+                          'data to be received from Marathon. Set to 0 to '
+                          'disable. (default: %(default)s)'),
+                    type=float,
+                    default=0)
```

Comment:
Maybe of note that I'm defaulting this to off so that this isn't a potentially breaking change.

Reply:
:+1: